lsst/ts/tunablelaser/settings.py

```diff
@@ -0,0 +1,45 @@
+import abc
```

Comment:
I don't think this classes are used anywhere. Do you really want them here?

Reply:
No.